.github/workflows/performance.yml

```diff
@@ -145,11 +143,11 @@ jobs:
                 return parseInt((new Date( commit_details.data.author.date ).getTime() / 1000).toFixed(0))
 
       - name: Publish performance results
+        if: github.event_name == 'push'
```

Comment:
Good point. I think for now we should ensure this only ever runs for `trunk`, not other point release branches.

But also I don't understand why this line here is being added. We do want to restrict pushing the metrics to Code Health to only the `trunk` branch for `push` events, but what value is there in running the rest of this workflow if that is not happening?

I would think we should rather control the entire workflow to only run on `push` events to `trunk`, not just this single step.

Reply:
Actually, never mind my feedback here, I now realize the `github.event_name = 'push'` here is right since we should also run the workflow on PRs, but not report the data to Code Health.

@joemcgill Do you think there is value of for now commenting out this line and putting a comment above to re-add it prior to merge? This way we can show that the implementation is working via the Code Health dashboard. Otherwise, it would only start sending data once merged I think.

Reply:
@joemcgill WFM.

I don't feel strongly about the idea of a "PR that isn't mergable". It will eventually be "manually" committed rather than just merged anyway, and I think calling out things like "Before committing, make change x" in bold on the PR description can be reasonable.

If you think our current test dashboard data is enough to show how this is working, let's go with that 👍